hera_cal/datacontainer.py

```diff
@@ -84,12 +92,81 @@ def __delitem__(self, key):
         else:
             raise ValueError('only supports setting (ant1, ant2, pol) keys')
 
+    def __xor__(self, D):
```

Comment:
Generally, operator overloading only happens when there's some notion of "addition" or whatever on the object. Concatenation across time is not obviously xor (if anything it's +). This notation is highly non-intutive and makes code very difficult to read.  You should make it `append(axis=0)` i.e. default axis is 0 (time), but it can also append along frequency.